packages/loader/container-loader/src/container.ts

```diff
@@ -2586,3 +2587,17 @@ export interface IContainerExperimental extends IContainer {
 	 */
 	closeAndGetPendingLocalState?(stopBlobAttachingSignal?: AbortSignal): Promise<string>;
 }
+
+/**
+ * @deprecated Will be removed in future major release.
+ * @alpha
+ */
+export async function requestOnIContainer_Deprecated(container: IContainer): Promise<FluidObject> {
+	const containerCast = container as Container;
+	if ("request" in containerCast.runtime && typeof containerCast.runtime.request === "function") {
+		const response: IResponse = await (containerCast.runtime as any).request({ url: "/" });
```

Comment:
At least as far as where Bohemia code would use this function -- [requestComponent.ts](https://office.visualstudio.com/OC/_git/office-bohemia?path=/packages/office-web-host/src/LoadHelpers/requestComponent.ts&version=GBmaster&line=140&lineEnd=141&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) -- there is a `request` input object. So I think this input needs to be plumbed through rather than `{ url: "/" }`, right?